src/sentry/migrations/0122_groupinbox_addprojandorg.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-11-05 16:10
+from __future__ import unicode_literals
+
+from django.db import migrations
+import django.db.models.deletion
+import sentry.db.models.fields.foreignkey
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = True
```

Comment:
With this being a dangerous migration, I don't think you should be adding the columns to the model in the same pull request. Including the model fields in this change will result in all queries failing between the deploy time and the time the migration completes.

Reply:
This PR actually has no migration. It was based on a different PR that had the migration and got merged. Merging this with master now and that should fix this.